holotrack.Tests/Visual/Tracking/TestSceneFaceTracking.cs

```diff
@@ -83,8 +84,33 @@ protected override void Update()
                     var index = faceLocationsContainer.IndexOf(box);
                     var face = faces[index];
 
-                    box.MoveTo(new Vector2(face.BoundingBox.X, face.BoundingBox.Y), box.FirstTrack ? 0 : 200, Easing.OutQuint);
-                    box.ResizeTo(new Vector2(face.BoundingBox.Width, face.BoundingBox.Height), box.FirstTrack ? 0 : 200, Easing.OutQuint);
+                    for(int i=0; i<box.Count; i++)
+                    {
+                        if(box.Children[i] is Circle)
+                        {
+                            box.Remove(box.Children[i]);
+                            i--;
+                        }
+                    }
+                    
+                    // TODO (AINL) : This should not be cropped by boundary of bounding box.
+                    foreach (var part in face.Landmarks.Values)
+                    {
+                        foreach (var point in part)
+                        {
+                            box.Add(new Circle
+                            {
+                                X = (float)point.Point.X - face.BoundingBox.X,
+                                Y = (float)point.Point.Y - face.BoundingBox.Y,
+                                Size = new Vector2(10),
+                                Colour = Colour4.Blue,
+                            });
+                        }
+                    }
+
+                    // TODO (AINL): This is working weird when tracking.
+                    box.MoveTo(new Vector2(face.BoundingBox.X, face.BoundingBox.Y), box.FirstTrack ? 0 : 0, Easing.OutQuint);
```

Comment:
Looks like this has some easing, @LeNitrous, I noticed this one working for some reason, maybe we can do something like this for the Live2D data?

Reply:
It looks weird if the animation turns on because the bounding box is moving very low precision while actual tracking points moving in high precision.

Reply:
You probably don't need to add Easing if that sacrifices the precision, we don't need easing for it anyways since it has to represent the precision of the Convolutional Network.